src/Components/Navigation/Navigation.scss

```
@@ -0,0 +1,695 @@
+@import "src/assets/stylesheets/bootstrap/mixins/text-hide";
+@import "src/assets/stylesheets/bootstrap/mixins/transition";
+
+/* .bg-dark-image .navbar-brand { background: $logo-white no-repeat center center / contain; } */
+.bg-dark-image {
+  .nav > li > a {
+    color: #fff !important;
+  }
+
+  .navbar-toggler .icon-bar {
+    background: #fff;
+  }
+}
+
+/* .scrolled.bg-dark-image .navbar-brand { background: $logo-dark no-repeat center center / contain; } */
+
+.scrolled {
+  .navbar {
+    background: rgba(255, 255, 255, 0.9);
+    box-shadow: 0 0 3px rgba(0, 0, 0, 0.2);
+  }
+
+  &.bg-dark-image {
+    .navbar-nav > {
+      li > a {
+        color: $theme-greydark !important;
+      }
+
+      li.active > a {
+        color: #fff !important;
+      }
+    }
+
+    .nav > li {
+      &.active > a,
+      &.open > a {
+        color: #fff !important;
+      }
+    }
+
+    .navbar-toggler .icon-bar {
+      background: $secondary;
+    }
+  }
+}
+
+.navbar {
+  padding: 0;
+  min-height: 75px;
+  box-shadow: 0 0 3px rgba(0, 0, 0, 0.2);
+
+  &-brand {
+    /* background: $logo-dark no-repeat center center / contain; */
+    width: 120px;
+    @include text-hide;
+    height: 75px;
+    padding: 25px 0;
+
+    img {
+      margin: 0;
+    }
+  }
+
+  &-nav > li > a {
+    height: 75px;
+    position: relative;
+    padding: 25px 15px !important;
+    text-transform: uppercase;
+    font-weight: 600;
+    font-size: 14px;
+    color: $theme-greydark;
+    display: inline-block;
+  }
+
+  &-search-toggle {
+    position: absolute;
+    top: 0;
+    right: 10px;
+    display: block;
+    height: 75px;
+    width: 50px;
+    padding: 23px 0;
+    text-align: center;
+
+    &:after {
+      font-size: 18px;
+      content: "|";
+      color: #ddd;
+      position: absolute;
+      top: 22px;
+      left: -9px;
+    }
+
+    &:hover {
+      cursor: pointer;
+    }
+
+    &.disabled {
+      &:hover {
+        cursor: auto;
+      }
+
+      .fa {
+        color: $btn-link-disabled-color;
+      }
+    }
+
+    .fa {
+      font-size: 22px;
+      padding: 0;
+      color: $secondary;
+    }
+  }
+
+  &-toggler {
+    float: left;
+    margin: 0;
+    padding: 9px 20px;
+    height: 75px;
+    border: none !important;
+    border-radius: 0;
+
+    &:hover,
+    &:focus {
+      background: transparent !important;
+    }
+
+    &:not(.collapsed) {
+      background: #fff !important;
+
+      .icon-bar {
+        background: $secondary !important;
+      }
+    }
+
+    .icon-bar {
+      display: block;
+      height: 2px;
+      border-radius: 1px;
+      background: $secondary;
+      width: 30px;
+
+      + .icon-bar {
+        margin-top: 6px;
+      }
+    }
+  }
+
+  &-transparent {
+    background: transparent;
+    @include transition(all 0.3s ease-in-out);
+    box-shadow: none;
+  }
+}
+
+.nav {
+  margin: 0;
+
+  &.navbar-right {
+    margin-left: auto;
+    margin-right: 45px;
+  }
+
+  > li {
```

If I'm not mistaken the following is missing (see my previous comment):

```suggestion
  > li,
  > li > a {
```